docs/sources/auth/auth-proxy.md

```diff
@@ -218,3 +218,18 @@ ProxyPassReverse / http://grafana:3000/
 ### Use grafana.
 
 With our Grafana and Apache containers running, you can now connect to http://localhost/ and log in using the username/password we created in the htpasswd file.
+
+### Team Sync (Enterprise only)
+
+>  Only available in Grafana Enterprise v6.3+
+
+With Team Sync you can feed team ID's to Grafana and map them to Teams in Grafana. This allows you to automatically put your users into specific teams. 
+
+To do this, you need to configure the optional `Groups` header and use that to send your team information.
+
+
+```bash
+# Example `headers = Groups:X-WEBAUTH-GROUPS`
+headers =
```

Comment:
this example is a bit strange, with the example in a comment and the uncomented line empty. Do we need to mention here how to get the team ids? is that Grafana internal team ids?

Reply:
Yeah, you're right. We should add more detail, showing real config and an example request using `curl` or such that uses the feature.

Reply:
```suggestion
headers = fuck-kde
```

Reply:
@torkelo could I please get a second look at this? I have added examples and reworded a the bit of auth proxy section.